Browser wasm on windows CI builds broken

While adding a new pipeline for the "extra" platforms (#62564) I've hit an issue with the browser wasm windows build legs:

```
HEAD is now at f44b841 2.0.23 (#828)
  Error: Downloading URL 'https://storage.googleapis.com/webassembly/emscripten-releases-builds/deps/node-v14.15.5-win-x64.zip': <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)>
  Warning: Possibly SSL/TLS issue. Update or install Python SSL root certificates (2048-bit or greater) supplied in Python folder or https://pypi.org/project/certifi/ and try again.
  Installation failed!
  Installing SDK 'sdk-releases-upstream-77b065ace39e6ab21446e13f92897f956c80476a-64bit'..
  Installing tool 'node-14.15.5-64bit'..
D:\a\_work\1\s\src\mono\mono.proj(153,5): error MSB3073: The command "git checkout 2.0.23 && powershell -NonInteractive -command "& ./emsdk.ps1 install 2.0.23; Exit $LastExitCode " && powershell -NonInteractive -command "& ./emsdk.ps1 activate 2.0.23; Exit $LastExitCode "" exited with code 1.
##[error]src\mono\mono.proj(153,5): error MSB3073: (NETCORE_ENGINEERING_TELEMETRY=Build) The command "git checkout 2.0.23 && powershell -NonInteractive -command "& ./emsdk.ps1 install 2.0.23; Exit $LastExitCode " && powershell -NonInteractive -command "& ./emsdk.ps1 activate 2.0.23; Exit $LastExitCode "" exited with code 1.
```

The build pipeline is: https://dev.azure.com/dnceng/public/_build/results?buildId=1559642&view=logs&jobId=854f6d42-496e-5e8c-a1c9-7f7194a94a83&j=bce77159-c177-5da3-02f5-74ccd320a9d5&t=55fcbaba-323b-59bd-3ce8-039d98ed8365

@radekdoulik can you please take a look? I will comment the build legs out with a reference to this issue.

--- a/eng/pipelines/common/global-build-job.yml
+++ b/eng/pipelines/common/global-build-job.yml
@@ -141,7 +141,7 @@ jobs:
           df -h
         displayName: Disk Usage before Build
 
-    - ${{ if contains(parameters.nameSuffix, 'Windows_wasm') }}:
+    - ${{ if eq(parameters.platform, 'Browser_wasm_win') }}:
       # Update machine certs
       - task: PowerShell@2
         displayName: Update machine certs

--- a/eng/pipelines/coreclr/templates/helix-queues-setup.yml
+++ b/eng/pipelines/coreclr/templates/helix-queues-setup.yml
@@ -21,6 +21,7 @@ jobs:
     archType: ${{ parameters.archType }}
     container: ${{ parameters.container }}
     pool: ${{ parameters.pool }}
+    platform: ${{ parameters.platform }}
     shouldContinueOnError: ${{ parameters.shouldContinueOnError }}
     dependOnEvaluatePaths: ${{ parameters.dependOnEvaluatePaths}}
     runtimeFlavorDisplayName: ${{ parameters.runtimeFlavorDisplayName }}

--- a/eng/pipelines/libraries/helix-queues-setup.yml
+++ b/eng/pipelines/libraries/helix-queues-setup.yml
@@ -21,6 +21,7 @@ jobs:
     archType: ${{ parameters.archType }}
     container: ${{ parameters.container }}
     pool: ${{ parameters.pool }}
+    platform: ${{ parameters.platform }}
     shouldContinueOnError: ${{ parameters.shouldContinueOnError }}
     dependOnEvaluatePaths: ${{ parameters.dependOnEvaluatePaths}}
     runtimeFlavorDisplayName: ${{ parameters.runtimeFlavorDisplayName }}

--- a/eng/pipelines/runtime-extra-platforms.yml
+++ b/eng/pipelines/runtime-extra-platforms.yml
@@ -568,8 +568,7 @@ jobs:
       condition: >-
         or(
           eq(dependencies.evaluate_paths.outputs['SetPathVars_wasmdebuggertests.containsChange'], true),
-          eq(variables['isManualOrIsNotPR'], true),
-          eq(variables['isFullMatrix'], true))
+          eq(variables['isRollingBuild'], true))
       # extra steps, run tests
       extraStepsTemplate: /eng/pipelines/libraries/helix.yml
       extraStepsParameters:

--- a/eng/pipelines/runtime-extra-platforms.yml
+++ b/eng/pipelines/runtime-extra-platforms.yml
@@ -130,8 +130,7 @@ jobs:
     platforms:
     # BuildWasmApps should only happen on the extra platforms build as we already have coverage for this build on PRs.
     - Browser_wasm
-    # disable until https://github.com/dotnet/runtime/issues/63987 is fixed
-    # - Browser_wasm_win
+    - Browser_wasm_win
     variables:
       # map dependencies variables to local variables
       - name: wasmbuildtestsContainsChange
@@ -169,8 +168,7 @@ jobs:
     buildConfig: release
     runtimeFlavor: mono
     platforms:
-    # disable until https://github.com/dotnet/runtime/issues/63987 is fixed
-    # - Browser_wasm_win
+    - Browser_wasm_win
     variables:
       # map dependencies variables to local variables
       - name: librariesContainsChange
@@ -257,8 +255,7 @@ jobs:
     runtimeFlavor: mono
     platforms:
     - Browser_wasm
-    # disable until https://github.com/dotnet/runtime/issues/63987 is fixed
-    # - Browser_wasm_win
+    - Browser_wasm_win
     variables:
       # map dependencies variables to local variables
       - name: librariesContainsChange
@@ -541,8 +538,7 @@ jobs:
       condition: >-
         or(
           eq(dependencies.evaluate_paths.outputs['SetPathVars_wasmdebuggertests.containsChange'], true),
-          eq(variables['isManualOrIsNotPR'], true),
-          eq(variables['isFullMatrix'], true))
+          eq(variables['isRollingBuild'], true))
       # extra steps, run tests
       extraStepsTemplate: /eng/pipelines/libraries/helix.yml
       extraStepsParameters:
@@ -559,8 +555,7 @@ jobs:
     buildConfig: Release
     runtimeFlavor: mono
     platforms:
-    # disable until https://github.com/dotnet/runtime/issues/63987 is fixed
-    # - Browser_wasm_win
+    - Browser_wasm_win
     variables:
       # map dependencies variables to local variables
       - name: wasmdebuggertestsContainsChange

